src/olympia/amo/tests/test_views.py

```diff
@@ -443,3 +445,27 @@ def test_version_json(self):
             sys.version_info.major, sys.version_info.minor)
         assert content['django'] == '%s.%s' % (
             django.VERSION[0], django.VERSION[1])
+
+
+class TestSiteStatusAPI(TestCase):
+    client_class = APITestClient
+
+    def setUp(self):
+        super().setUp()
+        self.url = reverse_ns('amo-site-status')
+
+    def test_response(self):
+        response = self.client.get(self.url)
+        assert response.status_code == 200
+        assert response.data == {
+            'read_only': False,
+            'notice': None,
+        }
+
+        set_config('site_notice', 'THIS is NOT Á TEST!')
+        with override_settings(READ_ONLY=True):
+            response = self.client.get(self.url)
+        assert response.data == {
+            'read_only': True,
+            'notice': 'THIS is NOT Á TEST!',
+        }
```

Comment:
Let's add two more test-cases that check for `READ_ONLY=False` but `site_notice` set and `READ_ONLY=True` but `site_notice` unset.

Reply:
https://github.com/mozilla/addons-server/pull/12356/commits/4acd93a499ade786efc4967b45807f2550d4a248